packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "styled-components/native";
+import Text from "../../../Text";
+import { TouchableOpacity } from "react-native";
+import TemplateToggleGroup, {
+  BaseToggleGroupProps,
+  ItemToggleGroupProps,
+} from "../TemplateToggleGroup";
+
+export const ToggleBox = styled(TouchableOpacity)<{ isActive: boolean }>`
+  text-align: center;
+  margin: auto;
+  flex: 1;
+  padding: ${(p) => p.theme.space[5]}px 0;
+  border-radius: 8px;
+  ${(p) =>
+    p.isActive &&
+    `
+    background-color: ${p.theme.colors.palette.primary.c20};
+  `}
```

Comment:
(optional/nitpicking): you can use 
`background-color: ${(p) => p.isActive ? p.theme.colors.palette.primary.c20 : 'unset'};`
here if you'd like, might save some lines

Reply:
I will use yours, it's way better than template inside template, thank you

Reply:
Little note about this change that i learnt and that could be useful for you too @gsoares-ledger  :smile: 
backgroundColor can't be 'unset' in react-native